### ISSUE TITLE
[Feature Request] Copy Code Snippet Button

### ISSUE THREAD

**Issue by aryanpingle** [GoogleChrome/developer.chrome.com#5296]
## **Is your feature request related to a problem? Please describe.**

The code snippet/examples given in the guides are fantastic. It would be super useful to have a way to copy the code snippets with a single button, rather than manually selecting the contents.

## **Describe the solution you'd like**

A simple "copy to clipboard" square icon-button on the top right of code snippets (like in markdown files on GitHub) would be great. It's a popular feature on several sites so it'll be convenient to have it here as well, and shouldn't be too difficult to implement.

## Relevant Screenshot

| ![snippet](https://user-images.githubusercontent.com/49340972/219045074-87757361-501d-45d4-ae73-5dee98179276.PNG) |
|:-:|
| Simple boilerplate code snippets like these are great, and there should be an easy way to copy the contents |

**Comment by MaAnCoSa** [comment]
Hi! Can I work on this issue?

**Comment by MaAnCoSa** [comment]
Hi! I made a PR and it was reviewed, but it didn't pass the lint test nor the Chrome DevRel PR checks. I fixed the lint errors and updated the PR, however I do not know what to do about the Chrome DevRel PR checks. I am still waiting on a new review.

**Comment by dstmarthe** [comment]
Hi, thanks for raising this. Looks like after the site migration all code snippets now have a copy code button.
If there are any other suggestions/issues you'd like to raise, you can visit the new issue tracker, specifically for web.dev, at 
https://issuetracker.google.com/issues?q=status:open%20componentid:1400680&s=created_time:desc
